src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -162,13 +162,15 @@ function MoneyRequestParticipantsSelector({participants = CONST.EMPTY_ARRAY, onF
         }
 
         const newOptions = OptionsListUtils.filterOptions(defaultOptions, debouncedSearchTerm, {
+            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+            canInviteUser: (canUseP2PDistanceRequests || iouRequestType !== CONST.IOU.REQUEST_TYPE.DISTANCE) && !isCategorizeOrShareAction,
```

Comment:
Is there a specific reason we're using logical OR (`||`) instead of nullish coalescing (`??`)?